webapp/components/wpt-amend-metadata.js

```diff
@@ -353,7 +367,38 @@ class AmendMetadata extends LoadingState(PathInfo(ProductInfo(PolymerElement)))
         node['label'] = '';
       }
       this.displayedMetadata.push(node);
+      this.fieldsFilled.filled.push(false);
     }
+    // A URL or label must be supplied for every triage item,
+    // which are all currently empty.
+    this.fieldsFilled.numEmpty = this.displayedMetadata.length;
+  }
+
+  handleFieldInput(event) {
+    // Detect which input was filled.
+    const index = event.model.__data.index;
+    const url = this.displayedMetadata[index].url;
+    const label = this.displayedMetadata[index].label;
+
+    // Check if the input is empty.
+    if (url === '' && (label === '' || label === undefined)) {
+      // If the field was previously considered filled, it's now empty.
+      if (this.fieldsFilled.filled[index]) {
+        this.fieldsFilled.numEmpty++;
+      }
+      this.fieldsFilled.filled[index] = false;
+    } else if (!this.fieldsFilled.filled[index]) {
+      // If the field was previously empty, it is now considered filled.
+      this.fieldsFilled.numEmpty--;
+      this.fieldsFilled.filled[index] = true;
+    }
+    this.checkEnableTriageButton();
+  }
+
+  checkEnableTriageButton() {
+    // If all triage items have input, triage can be submitted.
+    this.triageSubmitDisabled = (this.fieldsFilled.numEmpty > 0
+      || this.displayedMetadata.length === 0);
```

Comment:
`this.displayedMetadata.length === 0` isn't needed because user won't be able to enter the triage dialog if nothing is selected

Reply:
Makes sense - I added this 👌